src/main/less/core.less

```diff
@@ -97,7 +97,6 @@ svg:not(:root) {
         display: flex;
         align-items: center;
         margin: 0.5rem;
-        padding: 0.25rem 0.5rem;
```

Comment:
Why are we removing this? Will it affect any graphs?

Reply:
I have removed it from this file and added it to the constants.js file [here](https://github.com/cerner/carbon-graphs/pull/213/commits/616164b66e7a4f0ba408397f05c00fa7bf886de0#diff-965ff141b3e1c375076dd23d7144dd8aR158-R163). Moved it to the constants.js file so that we can access and set the legend padding defaults in getLegendPadding() [here](https://github.com/cerner/carbon-graphs/pull/213/commits/616164b66e7a4f0ba408397f05c00fa7bf886de0#diff-ed3a6d442fe463a74cc24a039542f917R67) when legendPadding is not provided or when legendPadding is only half provided.